rotkehlchen/db/dbhandler.py

```diff
@@ -233,7 +238,16 @@ def _check_unfinished_upgrades(self) -> None:
         if ongoing_upgrade_from_version is None:
             return  # We are all good
 
-        # Otherwise replace the db with a backup and reconnect
+        # if there is an unfinished upgrade, check user approval to resume from backup
+        if resume_from_backup is False:
+            raise RotkehlchenPermissionError(
+                error_message='The encrypted database is in a half-upgraded state. '
+                              'Either resume from a backup or solve the issue manually.',
```

Comment:
It's a pity we don't have a way of enforcing this via a linter but this is not how we handle multiline strings. Please do this instead:

Also please notice all the changes my diff had in the last review.

`half -> semi`

```suggestion
                error_message=(
                    'The encrypted database is in a semi upgraded state. '
                    'Either resume from a backup or solve the issue manually.'
                ),
```

or you can also just make it all ine one line if its' not too huge and add `#noqa: E501` at the end